Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1129,6 +1155,57 @@ def genS(fragment,howMuch):
 
 ###end UP17
 ###
+
+### FastSim: produce sample of premixed minbias events UP18
+steps["FS_PREMIXUP18_PU50"] = merge([
+        {"cfg":"SingleNuE10_cfi",
+         "--fast":"",
+         "--conditions":"auto:phase1_2018_realistic",
+         "-s":"GEN,SIM,RECOBEFMIX,DIGI",
+         "--eventcontent":"PREMIX",
+         "--datatier":"PREMIX",
+         "--procModifiers":"premix_stage1",
+         "--era":"Run2_2018_FastSim",
+         },
+        PUFSAVE50UP18,Kby(100,500)])
```

Comment:
It would be more maintainable to derive these from the configurations of earlier years instead of full copy-paste.

Reply:
unfortunately, for relval_steps.py, the maintainability train left the station many years ago...

Reply:
I know, but we could at least try to control the speed of the train.